apps/chat2/config_chat2.nim

```diff
@@ -180,7 +180,7 @@ type
 
     metricsLogging* {.
       desc: "Enable metrics logging: true|false"
-      defaultValue: false
```

Comment:
I also disagree that the console metrics logging task should be enabled by default.

Chat2 is a "toy app"/example, and we use it to do some sanity tests. I am okay with adding metrics support but not enabling it by default.

Reply:
Imo, we should enable it by default, because the next testnet would involve users/operators looking at proof verification times etc, and/or sending the metrics data to a central server for researchers to analyze. wdyt?